Sources/Blog/main.swift

```diff
@@ -11,12 +14,14 @@ struct Blog: Website {
 
     struct ItemMetadata: WebsiteItemMetadata {
         // Add any site-specific metadata that you want to use here.
+        var author: String
+        var leggings: String?
```

Comment:
Again left over

Reply:
the leggings is definitely left over, but the author i'm using in the vaportheme.swift to display it